modmail/__init__.py

```diff
@@ -54,4 +59,5 @@
 # Set asyncio logging back to the default of INFO even if asyncio's debug mode is enabled.
 logging.getLogger("asyncio").setLevel(logging.INFO)
 
-root.debug("Logging initialization complete")
+
+instance: "ModmailBot" = None  # Global ModmailBot instance.
```

Comment:
Shouldn't the correct annotation be `Optional["ModmailBot"]` since we set it to `None` here?